src/ZoneServer/Buffs/Handlers/DaggerSlash_Buff.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Melia.Zone.Buffs.Base;
+using Melia.Shared.Tos.Const;
+
+namespace Melia.Zone.Buffs.Handlers
+{
+	/// <summary>
+	/// Handler for DaggerSlash_Buff, which affects the character attack.
+	/// </summary>
+	[BuffHandler(BuffId.DaggerSlash_Buff)]
+	public class DaggerSlash_Buff : BuffHandler
+	{
+		/// <summary>
+		/// Starts buff, modifying the attack.
+		/// </summary>
+		/// <param name="buff"></param>
+		public override void OnStart(Buff buff)
+		{
+			var target = buff.Target;
+
+			// TODO: Something with the attack based on number of daggers equiped.
```

Comment:
I do not know how to get the number of daggers equiped yet so I will leave like that for now.

Reply:
The way I would interpret the skill description is that you always get the buff with a dagger equipped, but _if_ you have weapons equipped in the secondary slot you only get the bonus if both main weapons are daggers. Is that not how it works? If it is, it would be fine to implement it to always apply for now and come back to it later.

Regarding how to get the equipped items, you can simply query them from the inventory if the entity is a `Character`.
```cs
character.Inventory.GetEquip(EquipSlot.RightHand)
character.Inventory.GetEquip(EquipSlot.LeftHand)
character.Inventory.GetEquip(EquipSlot.RightHandSub)
character.Inventory.GetEquip(EquipSlot.LeftHandSub)
```